src/Core/Prefab/GlobeView.js

```diff
@@ -124,9 +124,7 @@ class GlobeView extends View {
             CameraUtils.transformCameraToLookAtTarget(this, this.camera.camera3D, placement);
         } else {
             this.controls = new GlobeControls(this, placement, options.controls);
-            this.controls.handleCollision = typeof (options.handleCollision) !== 'undefined' ? options.handleCollision : true;
```

Comment:
I shouldn't have deleted this line because some examples are putting `handleCollision` parameters directly in view options instead of options.controls. I revert this change